Documentation/ApiOverview/DirectoryStructure/LegacyInstalations.rst

```diff
@@ -0,0 +1,117 @@
+..  include:: /Includes.rst.txt
+..  index::
+    Path; Legacy installations
+    see: Directory structure; Path
+..  _legacy-directory-structure:
+
+=========================================
+Legacy installations: Directory structure
```

Comment:
The file name should be adjusted to "LegacyInstallation" (double L).

Also, I suggest to not add a separate file for legacy, but integrate it on the index page. Maybe the differences just beyond the Composer-based folders.

This way we don't have to maintain two files and users looking for the structure via search engine get on main result, not two. I think, two files may confuse users which just scan the page.

So, I haven't reviewed this page for now.

Reply:
There are really no two directories with the same position as there is no public folder. I also kept it short to the minimum and then linked to other chapters.  It would only get more confusing if we only mention some chapters here.